daal4py/sklearn/test/test_patching/test_patching.py

```diff
@@ -0,0 +1,220 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+def make_unique(mas):
+    s = set()
+    ans = []
+    for i in mas:
+        if i not in s:
+            ans.append(i)
+            s.add(i)
+    return ans
+
+
+def get_method(s):
+    cnt = 0
+    ind = -1
+    for i, elem in enumerate(s):
+        if elem == '.':
+            cnt += 1
+        if cnt == 3:
+            ind = i
+            break
+    assert ind != -1, "Bad string"
+    method = ''
+    for i in range(ind + 1, len(s)):
+        if s[i] == ':':
+            break
+        method += s[i]
+    return method
+
+
+def get_branch(s):
+    if len(s) == 0:
+        return 'NO INFO'
+    for i in s:
+        if 'uses original Scikit-learn solver,' in i:
+            return 'was in IDP, but go in Scikit'
+    for i in s:
+        if 'uses Intel(R) oneAPI Data Analytics Library solver' in i:
+            return 'IDP'
```

Comment:
IDP - Intel® Distribution for Python
I propose an other name, for example OPT, optimize, etc

```suggestion
            return 'OPT'
```